x509util/files.go

```diff
@@ -52,10 +52,10 @@ func ReadPossiblePEMURL(target, blockname string) ([][]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to ioutil.ReadAll(%q): %v", target, err)
 	}
-	return dePEM(data, blockname), nil
+	return DePEM(data, blockname), nil
 }
 
-func dePEM(data []byte, blockname string) [][]byte {
+func DePEM(data []byte, blockname string) [][]byte {
```

Comment:
Please add a comment since this func is now exported.

Reply:
Done.